.redhat-ci.yaml

```diff
@@ -6,59 +6,10 @@
 
 # REQUIRED (only one of 'host' or 'container' or 'cluster')
 # Provision a single host.
-host:
+container:
     # REQUIRED
-    # Specify the distro to provision:
-    #   - fedora/24/cloud
-    #   - fedora/24/atomic
-    #   - fedora/25/cloud
-    #   - fedora/25/atomic
-    #   - fedora/latest/atomic
-    #   - centos/7/cloud
-    #   - centos/7/atomic
-    #   - centos/7/atomic/alpha
-    #   - centos/7/atomic/continuous
-    distro: fedora/25/cloud
-
-    # OPTIONAL
-    # Specify the minimum requirements the host must
-    # fulfill. The values defined below are the defaults
-    # used when no minimums are specified.
-    specs:
-      # OPTIONAL
-      # Minimum amount of RAM in MB.
-      ram: 2048
-      # OPTIONAL
-      # Minimum number of CPUs.
-      cpus: 1
-      # OPTIONAL
-      # Minimum size of primary disk in GB.
-      disk: 20
-      # OPTIONAL
-      # Minimum size of secondary disk in GB. Set to 0 if no
-      # secondary disk is needed.
-      secondary-disk: 0
-
-    # OPTIONAL
-    # If using an Atomic Host variant, you can specify which
-    # ostree to use. To use the latest tree on the current
-    # branch, use:
-    ostree: latest
-
-    # Otherwise, you may specify a tree by providing a dict:
-    ostree:
-      # OPTIONAL
-      # Specify the remote to which to rebase. If omitted,
-      # the current remote is used.
-      remote: http://example.com/remote/repo
-      # OPTIONAL
-      # Specify the branch to which to rebase. If omitted,
-      # the current branch is used.
-      branch: my/branch
-      # OPTIONAL
-      # Specify the version or checksum to deploy. If
-      # omitted, the latest commit is used.
-      revision: 7.145.42
+    # Specify an FQIN or Docker Hub image.
+    image: fedora:25
```

Comment:
FWIW I always use `registry.fedoraproject.org/fedora:25` now to avoid dependence on the Docker Hub.